src/nrf_802154.h

```diff
@@ -1068,7 +1104,15 @@ void nrf_802154_auto_pending_bit_set(bool enabled);
 bool nrf_802154_pending_bit_for_addr_set(const uint8_t * p_addr, bool extended);
 
 /**
- * @brief Removes address of a peer node for which there is no more pending data in the buffer.
+ * @brief Removes address of a peer node from the pending bit list.
+ *
+ * Pending bit list works differently for Thread and Zigbee applications.

```

Comment:
I think this comment, common for all the functions, would sound better if it didn't restrict to Thread and Zigbee applications only. I mean something like (just an idea):
`Pending bit list works differently based on the upper layer protocol used.`

And then the three currently supported cases can be listed as it is now.